### DISCUSSION TITLE
QUESTION: How to update Railway deployment?

### DISCUSSION THREAD

**Discussion by mayurjobanputra** [danny-avila/LibreChat#2131] [Q&A] [answered]
How do I update my installation of Librechat that was installed using one-click Railway app install? Do I use the "redeploy" option in Railway?

![image](https://github.com/danny-avila/LibreChat/assets/6332663/da282b20-abec-4b28-9cc7-83f69d562559)

**Comment by berry-13** [comment, marked as answer]
yes, with redeploy, it will pull the latest image

https://discord.com/channels/1086345563026489514/1219406429333487616
